python_modules/dagster-airflow/dagster_airflow_tests/test_dagster_plugin.py

```diff
@@ -0,0 +1,92 @@
+import os
+
+import pytest
+
+from airflow.exceptions import AirflowException
+
+from dagster_airflow.dagster_plugin import ModifiedDockerOperator
```

Comment:
should we be including DagsterOperator instead of ModifiedDockerOperator?

Reply:
yep